pixi.toml

```diff
@@ -56,13 +56,13 @@ rerun-web = { cmd = "cargo run --package rerun-cli --no-default-features --featu
 #
 # This installs the `wasm32-unknown-unknown` rust target if it's not already installed.
 # (this looks heavy but takes typically below 0.1s!)
-rerun-build-web = "rustup target add wasm32-unknown-unknown && cargo run -p re_dev_tools -- build-web-viewer --debug"
+rerun-build-web = "rustup target add wasm32-unknown-unknown && cargo run -p re_dev_tools -- build-web-viewer"
```

Comment:
the `--debug` flag is no longer supported. you opt-in to `--release` like you would on cargo
(this is because this was easier to do in `argh`. This not being fixed already was an oversight in the re_dev_tools pr)

Reply:
I prefer the old behavior; explicit over implicit. Otherwise it is too easy to do this:

`cargo run -p re_dev_tools --release -- build-web-viewer`

and then be very surprised by the wasm being a debug build

Reply:
kk, filed a ticket here, likely gonna go after it later today. Ofc this is out of scope of this pr https://github.com/rerun-io/rerun/issues/6014

Reply:
I think it's actually very good that it complains about unknown flags. Like I really want to know when I typoed on a flag instead of having it do nothing